src/rdb.cpp

```diff
@@ -2184,6 +2187,14 @@ void rdbLoadProgressCallback(rio *r, const void *buf, size_t len) {
         loadingProgress(r->processed_bytes);
         processEventsWhileBlocked(serverTL - g_pserver->rgthreadvar);
         processModuleLoadingProgressEvent(0);
+
+        robj *ping_argv[1];
+
+        ping_argv[0] = createStringObject("PING",4);
+        replicationFeedSlaves(g_pserver->slaves, g_pserver->replicaseldb, ping_argv, 1);
+        decrRefCount(ping_argv[0]);
+
+        r->loaded_keys = 0;
```

Comment:
Its rather confusing that this doesn't really track the number of loaded keys, but the keys since we last processed an event.  Can you fix the name.